content/doc/developer/forms/jelly-form-controls.adoc

```diff
@@ -167,7 +167,9 @@ Will automatically display the help button with the `<div>...</div>` content.
 
 * Most controls support `help.html` as overall help for the `Describable`.
 
-* The help message can be overridden in jelly with the `help` attribute, but please use the convention 'help-fieldName.html' as much as possible.
+* The help message can be overridden in jelly with the `help` attribute, but please use the convention `help-fieldName.html` as much as possible.
+
+* It is also possible to use localized help files, with the language specific suffix, so `help-fieldName.html` gets `help-fieldName_de.html` for the german version.
```

Comment:
I think you can also use jelly files for the help which would make more sense as then you you just modify the properties rather than the whole html file?

(non blocking)

Reply:
Not sure, some of the help files tend to have a lot of content, but I can also add this possibility.